wasmer/global.go

```diff
@@ -33,6 +35,40 @@ func (self *Global) ownedBy() interface{} {
 	return self._ownedBy
 }
 
+func (self *Global) Set(value interface{}, kind ValueKind) error {
+	value, err := fromGoValue(value, kind)
+	wasmValue := value.(C.wasm_val_t)
+
+	if err != nil {
+		//TODO: Make this error explicit
+		panic(err.Error())
+	}
+
+	C.wasm_global_set(self.inner(), &wasmValue)
```

Comment:
[Our implementation will panic](https://github.com/wasmerio/wasmer/blob/aeea0d6b546ce3ec76f7f45b976ff3623a24f02d/lib/c-api/src/wasm_c_api/externals/global.rs#L51) (with `unwrap`) in case of an error. That's probably something we want to fix! Can you submit a patch to fix that on `wasmerio/wasmer` please?